src/modules/user/dto/create-user.dto.ts

```diff
@@ -1,35 +1,72 @@
 import { ApiProperty } from '@nestjs/swagger';
-import { IsNotEmpty } from 'class-validator';
+import { IsArray, IsNotEmpty } from 'class-validator';
 import { Schema } from 'mongoose';
 
 export class CreateUserDTO {
   /**
-   * First name of user
-   * @example John
+   * First name of the user
+   * @example 'John'
    */
   @IsNotEmpty()
   readonly first_name: string;
 
+  /**
+   * Last name of the user
+   * @example 'Stark'
+   */
   readonly last_name: string;
 
+  /**
+   * Email of the user
+   * @example 'John@gmai.com'
+   */
   @IsNotEmpty()
   readonly email: string;
 
+  /**
+   * Phone number of the user
+   * @example '9000500000'
+   */
   @IsNotEmpty()
   readonly phone: string;
 
+  /**
+   * Address of the user
+   * @example 'Mayur Vihar, Delhi'
+   */
   readonly address: string;
 
+  /**
+   * Description of the user
+   * @example 'Aspiring Software Developer'
+   */
   readonly description: string;
 
+  /**
+   * Number of user
+   * @example 100
+   */
   readonly score: number;
 
+  /**
+   * The field to show whether the students is Admin or not
+   * @example false
```

Comment:
Also add @default preperty here which is `false`

Reply:
ok